src/components/Select/Select.stories.mdx

```diff
@@ -39,198 +38,214 @@ import { getCategory } from "../../utils/componentCategories";
 | Component Version | DS Version |
 | ----------------- | ---------- |
 | Added             | `0.7.0`    |
-| Latest            | `0.24.0`   |
+| Latest            | `0.24.2`   |
 
 <Description of={Select} />
 
-The `Select` component renders a `select` element along with its `option` children. For optimal accessibility, the `labelText` property is a required prop, regardless of the label visibility. Additionally, while the `id` prop is optional, a unique `id` attribute is necessary for accessibility. If the prop is left blank, a value will be generated for you.
+The `Select` component renders a `select` element along with its `option`
+children. For optimal accessibility, the `labelText` property is a required
+prop, regardless of the label visibility. Additionally, while the `id` prop is
+optional, a unique `id` attribute is necessary for accessibility. If the prop
+is left blank, a value will be generated for you.
 
-<Preview withToolbar>
+The `Select` component renders all the necessary wrapping components but the
+children _need_ to be `option` HTML elements.
```

Comment:
I was not sure what this meant at first.  Can you adjust the language to something like this?
```
The `Select` component renders all the necessary wrapping and associated text elements, but the select options _need_ to be child `<option>` HTML elements.
```